flags.css

```diff
@@ -1,5 +1,14 @@
 /* Indonesia */
-
+.flagContainer {
+    width: 50vw;
+    height: 50vh;
+    border: solid black;
+}
+.red {
```

Comment:
since this is more than just red, I recommend making the name more expressive as to what it is.  It is a half height element.  And it is red.  Maybe it is a indonesiaRedPanel?

Also you don't need width 100%, because is that by default